src/pkgdev/tatt/template.sh.jinja

```diff
@@ -0,0 +1,123 @@
+{#
+Copyright (C) 2010-2022 Gentoo tatt project
+https://gitweb.gentoo.org/proj/tatt.git/
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License
+as published by the Free Software Foundation; either version 2
+of the License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+#}
+#!/bin/bash
```

Comment:
Must this be bash?  My rough thought here was a python script that had the argument list just json blobbed into it, and then it feeds that back to the pkgdev_tatt code.

Pardon if that's assinine, I'm just going through the code and learning. :)

Reply:
My idea is to create a good replacement for `tatt` tool, and not a full wrapper. Folks can use other jinja templates (`--template-file`) to create a whole other possibilities. For a more sophisticated tool, like a testing server, maybe this more inline with [tattoo](https://gitlab.gentoo.org/arthurzam/tattoo).